akka-http-core/src/main/scala/akka/http/impl/settings/RoutingSettingsImpl.scala

```diff
@@ -17,7 +17,7 @@ private[http] final case class RoutingSettingsImpl(
   rangeCountLimit:          Int,
   rangeCoalescingThreshold: Long,
   decodeMaxBytesPerChunk:   Int,
-  fileIODispatcher:         String) extends akka.http.scaladsl.settings.RoutingSettings {
+  fileIODispatcher:         String  = "") extends akka.http.scaladsl.settings.RoutingSettings {
```

Comment:
can be made a method instead in order to keep compatibility; also mark it as deprecated then